datafusion/core/tests/dataframe/mod.rs

```diff
@@ -210,7 +210,6 @@ async fn test_count_wildcard_on_aggregate() -> Result<()> {
     let sql_results = ctx
         .sql("select count(*) from t1")
         .await?
-        .select(vec![count(wildcard())])?
```

Comment:
Select an aggregation expression on top of a Projection plan should be invalid.

Reply:
I agree that this tests is not a valid.

Perhaps it meant to show how to select the count(*) again -- if so maybe we could update the tests to do  something like this (rather than removing it entirely):

```
        .select(vec![col("COUNT(*)")])?
```

Reply:
Fixed. Thank you @alamb